docs/markdown/Trial-conversions.md

```diff
@@ -20,3 +20,14 @@ Here is a list of projects that have had a build system port done.
 ## More from WrapDB
 
 The [Meson Wrap database](https://wrapdb.mesonbuild.com) contains several projects that have been converted into Meson and are ready to use.
+
+# Other projects ported to Meson
+
+Other projects have got a Meson build configured within the project [Lhelper](https://github.com/franko/lhelper).
+
+Here is a list of projects that have had a build system port done within Lhelper.
```

Comment:
OK, but these are not _projects_ that are ported, just examples of doing the port.

Reply:
Ok, but I am not sure what do you mean. When you say "these" do you mean the project already listed or my additions ?

Do you mean that my addition should not be added in this page but maybe somewhere else ?

Reply:
There are tens or dozens of package managers and library provider libraries. We have not added links to any one of them, because then we should add all of them and it gets unwieldy quite quickly.

If you have ported libraries over to Meson we would appreciate if you could submit them to wrapdb instead. This way everyone can use them directly.

Reply:
That's ok, thank you, I am removing the pull request.